### ISSUE TITLE
Ruby integration example

### ISSUE THREAD

**Issue by ShockwaveNN** [codecov/codecov-action#62]
So I trying to replace old codecov integration in my projects (which worked fine) with Github Actions integration

My `ruby.yml` file looks like this:
```
name: Ruby
on: [push]
jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        ruby: [2.4, 2.5, 2.6, 2.7, head]
    steps:
    - uses: actions/checkout@v2
    - name: Set up Ruby ${{ matrix.ruby }}
      uses: ruby/setup-ruby@v1
      with:
        ruby-version: ${{ matrix.ruby }}
    - name: Build and test with Rake
      run: |
        gem install bundler
        bundle config set without 'development'
        bundle install --jobs 4 --retry 3
        bundle exec rake
    - name: Send test coverage report to codecov.io
      uses: codecov/codecov-action@v1
      with:
        file: ./coverage/.resultset.json
```

There is no error in logs:
![image](https://user-images.githubusercontent.com/668524/76527120-bcd63300-647f-11ea-9b66-48ac746ab037.png)

But after a go to codecov here:
https://codecov.io/gh/ShockwaveNN/pocket-sinatra-interface/commits
All I see this error:
![image](https://user-images.githubusercontent.com/668524/76527168-d5dee400-647f-11ea-9f05-006a1b9703aa.png)

Can you point out that I'm doing wrong?

**Comment by ShockwaveNN** [comment]
This is PR for this feature I'm trying to implement https://github.com/ShockwaveNN/pocket-sinatra-interface/pull/68

**Comment by Drowze** [comment]
I've also ran into this issue. It seems that this action does not support simplecov 0.18. I had success using 0.17 instead:

See:
https://github.com/Drowze/aspirebudgeting_ruby/pull/1/commits/8ad0f0636cc86440aca5af33b977542d8d69a2f4
https://github.com/Drowze/aspirebudgeting_ruby/runs/856078634

**Comment by thomasrockhu** [comment]
@Drowze sorry for the delay here, can you point me to a build where `0.18` was breaking?

**Comment by Drowze** [comment]
@thomasrockhu I was going to point you to this old build, but since it seems that it got deleted (the branch it was in rebased and push forced).
https://github.com/Drowze/aspirebudgeting_ruby/actions/runs/163053732

I used the same commit (with a dummy commit on top of it generate a new commit SHA) and pushed a new PR with it:
https://github.com/Drowze/aspirebudgeting_ruby/pull/10
https://github.com/Drowze/aspirebudgeting_ruby/runs/926732365
https://codecov.io/github/Drowze/aspirebudgeting_ruby/commit/ee483b984effe2e08dbacc58131532b84bf19e56

Please let me know if I can help any further.

**Comment by thomasrockhu** [comment]
Awesome, thanks @Drowze. We'll work on a fix for this.

**Comment by phosphene** [comment]
Thanks gents. I am also waiting on support for simplecov 0.18+ .  I downgraded to 0.17.1 in the meantime and it works quite well.

**Comment by thomasrockhu** [comment]
@phosphene @Drowze we made some changes on our side to support Simplecov 0.18+. Would you be able to see if this aligns with what you expect?

**Comment by pioz** [comment]
I have the same problem with version `0.18.5` (last that support Ruby 2.4). Here my config:
```
- name: Upload coverage report to Codecov
        uses: codecov/codecov-action@v1
        with:
          file: ./coverage/.resultset.json
```

![screnshot1](https://imgur.com/lXNZV8h.png)
![screnshot1](https://imgur.com/u1xzNvb.png)
![screnshot1](https://imgur.com/TLFv8eB.png)
